src/common/net.spec.ts

```diff
@@ -26,7 +26,9 @@ describe('Net', () => {
       });
     });
 
-  describe('#isPortAvailable', () => {
+  (process.platform === 'linux' && process.arch === 'arm64'
+    ? describe.skip
+    : describe)('#isPortAvailable', () => {
```

Comment:
yak shave - maybe due to node slim image, need to try with full fat.

Reply:
We're best keeping an eye on this, networking related bugs are a pain.

Reply:
110%

Reply:
My guess is that it's more to do with what's running/available on the Cirrus CI environment, than anything arm64 specific. But it is a reminder of the unfortunate unreliability of that feature. I'm actually wondering if we need to deprecate/drop support for it, as in the current state it is not a straightforward x-platform feature.

Reply:
So I had a look at the failing test. It's failing, because we're expecting it _not_ to be able to find a free port on port `80`. But if it's running in a container (like Cirrus CI, I believe) then port `80` may well be allowed, and the test could pass. 

I'll see if we can ensure it's "unavailable" by other means.

Reply:
oh yes I see now that on windows it sets `-1`, other platforms `80`. 

yes cirrus-ci in our current config is using a Dockerfile (or image) as the CI environment